helm_deploy/values-dev.yaml

```diff
@@ -23,4 +23,5 @@ env:
   EMAIL_LOCATION_URL: https://dev.use-of-force.service.justice.gov.uk
   TOKENVERIFICATION_API_URL: https://token-verification-api-dev.prison.service.justice.gov.uk
   TOKENVERIFICATION_API_ENABLED: true
+  HASH_SECRET: 'someHashSecret'
```

Comment:
I think HASH_SECRET should be in _envs.tpl not here.
Perhaps something like
```
  - name: HASH_SECRET
    valueFrom:
      secretKeyRef:
        name: {{ template "app.name" . }}
        key: HASH_SECRET
```
Then the secret needs adding to the application's kubernetes secrets.
HASH_SECRET is a bit vague as a name. URL_SIGNING_SECRET perhaps?

Reply:
(my fault on the rubbish name!)

Reply:
done

Reply:
done